lib/cb/clients/recommendation.rb

```diff
@@ -65,12 +65,15 @@ def self.for_user(*args)
       def self.for_company(company_did)
         my_api = Cb::Utils::Api.instance
         json_hash = my_api.cb_get(Cb.configuration.uri_recommendation_for_company, query: { CompanyDID: company_did })
-
         jobs = []
         if json_hash.key?('Results')
           if json_hash['Results'].key?('JobRecommendation')
-            json_hash['Results']['JobRecommendation']['Jobs'].each do |cur_job|
-              jobs << Models::Job.new(cur_job)
+            if json_hash['Results']['JobRecommendation'].key?('Jobs')
+              if json_hash['Results']['JobRecommendation']['Jobs'].key?('CompanyJob')
+                json_hash['Results']['JobRecommendation']['Jobs']['CompanyJob'].each do |cur_job|
+                  jobs << Models::Job.new(cur_job)
+                end
+              end
             end
```

Comment:
What would you think about using the `fetch` command instead of chaining all these if statements? Something like: `json_hash.fetch('Results', {}).fetch('JobRecommendation', {}).fetch('Jobs', {}).fetch('CompanyJob', []).each do |cur_job|`